x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning.tsx

```diff
@@ -34,40 +40,195 @@ interface Props {
 }
 
 export const RelevanceTuning: React.FC<Props> = ({ engineBreadcrumb }) => {
-  const { initializeRelevanceTuning } = useActions(RelevanceTuningLogic);
+  const { resetSearchSettings, initializeRelevanceTuning, updateSearchSettings } = useActions(
+    RelevanceTuningLogic
+  );
+  const { schemaFieldsWithConflicts, engineHasSchemaFields, unsavedChanges } = useValues(
+    RelevanceTuningLogic
+  );
+  const {
+    engine: { invalidBoosts, unsearchedUnconfirmedFields },
+  } = useValues(EngineLogic);
+
+  const schemaFieldsWithConflictsCount = schemaFieldsWithConflicts.length;
 
   useEffect(() => {
     initializeRelevanceTuning();
   }, []);
 
   return (
     <>
+      <UnsavedChangesPrompt hasUnsavedChanges={unsavedChanges} />
       <SetPageChrome trail={[...engineBreadcrumb, RELEVANCE_TUNING_TITLE]} />
-      <EuiPageHeader>
-        <EuiPageHeaderSection>
-          <EuiTitle size="l">
-            <h1>{RELEVANCE_TUNING_TITLE}</h1>
-          </EuiTitle>
-          <EuiText>
-            <EuiTextColor color="subdued">
+      <EuiPageHeader
+        className="relevanceTuningHeader"
+        pageTitle={RELEVANCE_TUNING_TITLE}
+        description={i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.description',
+          {
+            defaultMessage: 'Set field weights and boosts',
+          }
+        )}
+        rightSideItems={
+          engineHasSchemaFields
+            ? [
+                <EuiButton
+                  data-test-subj="SaveRelevanceTuning"
+                  color="primary"
+                  fill
+                  onClick={updateSearchSettings}
+                >
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.saveButtonLabel',
+                    {
+                      defaultMessage: 'Save',
+                    }
+                  )}
+                </EuiButton>,
+                <EuiButton
+                  data-test-subj="ResetRelevanceTuning"
+                  color="danger"
+                  onClick={resetSearchSettings}
+                >
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.resetButtonLabel',
+                    {
+                      defaultMessage: 'Restore defaults',
+                    }
+                  )}
+                </EuiButton>,
+              ]
+            : []
+        }
+      />
+
+      <EuiSpacer />
```

Comment:
Unfortunately this is stacking double `EuiSpacer`s (with the one below the conditional callouts) for tuning pages with no callouts and looks fairly odd.

<img width="588" alt="" src="https://user-images.githubusercontent.com/549407/109219789-ef4e6e00-776c-11eb-98c1-fe59fec312d7.png">

I think I'd vote for just deleting it, I don't think we need the extra negative space here

```suggestion
```